test/check-multi-machine

```diff
@@ -87,6 +87,11 @@ class TestMultiMachine(MachineCase):
         self.machine3.wait_boot()
         self.machine3.execute("hostnamectl set-hostname machine3")
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1297745
+        if self.machine.image == "rhel-atomic":
```

Comment:
There is a lot of fallout from the non-writable /tmp and we would have many known issues.  (Maybe 5 or 6?)  I would argue that this isn't a work around, but the real fix (just in the wrong place), and with it in place we can already now run more of our tests.